shavar/__init__.py

```diff
@@ -53,12 +54,23 @@ def get_configurator(global_config, **settings):
     return config
 
 
+def group_errors(event, hint):
+    if 'logentry' in event and 'message' in event['logentry']:
+        message = event['logentry']['message']
+        pattern = 'Invalid RANGE "[0-9]{10}'
+        if re.match(pattern, message) is not None:
+            event['fingerprint'] = ['invalid-range']
```

Comment:
@jwhitlock, any suggestion on the naming for fingerprint? I was wondering including `burp-invalid-range` to indicate that these are events caused by `burpsuite`. Wdyt?

Reply:
either `invalid-range` or `burp-invalid-range` would be fine with me